include/enums.h

```diff
@@ -1606,6 +1607,7 @@ enum NpcFlags {
 enum PlayerAnimFlags {
     PLAYER_ANIM_FLAG_HOLDING_ITEM    = 0x00000001,
     PLAYER_ANIM_FLAG_2               = 0x00000002,
+    PLAYER_ANIM_FLAG_3               = 0x00000003,
```

Comment:
since this is a flag, it's actually 2 | 1, so instead of a "3" flag you would just do `(PLAYER_ANIM_FLAG_HOLDING_ITEM | PLAYER_ANIM_FLAG_2)`